DEVELOPER_GUIDE.md

```diff
@@ -39,7 +49,15 @@ node.max_local_storage_nodes: 3
 ######## End OpenSearch Security Demo Configuration ########
 ```
 
-Next, we need to check out OpenSearch Dashboards from the branch matching our version, in this example that would be the [1.x branch](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/1.x). Follow the [developer guide](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/1.x/DEVELOPER_GUIDE.md) and replace the version of  `opensearch-dashboards.yml` there with this:
+
+**Please Note** : This project runs on node `10.24.1` and so when installing node please ensure that you install this version. You can do so by running 
```

Comment:
I'd probably make this relative to `.node-version` so we don't have to upgrade every time (I'm not sure if we already use a `.node-version` file but we should)

Reply:
ahaa...yes...we do use `.node-version` and `.nvm-rc`.. I just added an extra explanation saying "refer to .node-version file in base directory"